client/src/DualChart.jsx

```diff
@@ -0,0 +1,82 @@
+import React, { useEffect, useState } from 'react';
+import axios from 'axios';
+import { Dropdown } from 'primereact/dropdown';
+import ReactECharts from 'echarts-for-react';
+
+function DualChart() {
+    const [series, setSeries] = useState('CPIAUCSL');
+    const [dataPoints, setDataPoints] = useState([]);
+    const [loading, setLoading] = useState(false);
+
+    const seriesOptions = [
+        { label: 'Consumer Price Index (CPI)', value: 'CPIAUCSL', description: 'Measures changes in the price level of a basket of consumer goods and services.' },
+        { label: 'Unemployment Rate', value: 'UNRATE', description: 'Represents the percentage of the total labor force that is unemployed but actively seeking employment.' },
+        { label: 'Real GDP', value: 'GDPC1', description: 'Tracks the value of all goods and services produced, adjusted for inflation.' },
+        { label: 'Federal Funds Rate', value: 'FEDFUNDS', description: 'Interest rate at which depository institutions lend balances at the Federal Reserve to other institutions overnight.' }
+    ];
+
+    const selectedMeta = seriesOptions.find(s => s.value === series);
+
+    useEffect(() => {
+        const fetchData = async () => {
+            setLoading(true);
+            try {
+                const res = await axios.get(`http://localhost:8000/api/v1/fred/observations/${series}?sort_order=asc`);
```

Comment:
```suggestion
                const res = await axios.get(`${API_BASE_URL}/api/v1/v1/fred/observations/${series}?sort_order=asc`);
```
Hey you miss this one